src/Pdf/CakePdf.php

```diff
@@ -227,6 +227,8 @@ class CakePdf
         'fill_in'
     ];
 
+    protected $_mode = 'th';
```

Comment:
Will `th` might be a good default for you it might not be for everyone. Either keep it null or set to `'utf-8'`.

Reply:
Agreed 👍